src/hooks/api/organizers.ts

```diff
@@ -149,18 +138,29 @@ const useGetOrganizerByIdQuery = (
     ...configuration,
   });
 
-type GetOrganizersByCreator = HeadersAndQueryData;
+type GetOrganizersByCreator = { headers: Headers } & {
+  q: string;
+  limit: string;
+  start: string;
+  embed: string;
+};
```

Comment:
Is there a reason why you write as 2 objects?
And not in 1? 
```suggestion
type GetOrganizersByCreator = { 
  headers: Headers;
  q: string;
  limit: string;
  start: string;
  embed: string;
};
```

Reply:
In this case it is possible but in other places we reuse the type without the headers. Because the headers only get injected when using fetchFromApi